apps/extension/src/popup.ts

```diff
@@ -45,7 +53,9 @@ const spawnDetachedPopup = async (path: string) => {
 const throwIfAlreadyOpen = (path: string) =>
   chrome.runtime
     .getContexts({
-      documentUrls: [chrome.runtime.getURL(path)],
+      documentUrls: [
+        path.startsWith(chrome.runtime.getURL('')) ? path : chrome.runtime.getURL(path),
```

Comment:
What's this do? / why this change?

Reply:
left a pr comment here that may have been concealed by gh - the previous logic was failing to match paths because it would unnecessarily append an extension origin to a path that already contained it